gazelle/python/std_modules.go

```diff
@@ -25,19 +25,18 @@ import (
 	"strconv"
 	"strings"
 	"sync"
-	"time"
 
 	"github.com/bazelbuild/rules_go/go/tools/bazel"
 )
 
 var (
-	stdModulesStdin  io.Writer
+	stdModulesStdin  io.WriteCloser
```

Comment:
Why this? You should not close stdio.

Reply:
According to the official doc of [StdinPipe](https://pkg.go.dev/os/exec#Cmd.StdinPipe):

> A caller need only call Close to force the pipe to close sooner. For example, if the command being run will not exit until standard input is closed, the caller must close the pipe.

Both parser.py and std_modules.py loop through their stdin until EOF. So the proper way to shutdown those two processes is to send them EOF by closing the pipe:

https://github.com/bazelbuild/rules_python/blob/0d59fcf561f6d2c4705924bc17c151fb4b998841/gazelle/python/parse.py#L84

https://github.com/bazelbuild/rules_python/blob/0d59fcf561f6d2c4705924bc17c151fb4b998841/gazelle/python/std_modules.py#L39

Reply:
I see, thanks.